src/prettytable/prettytable.py

```diff
@@ -337,39 +337,39 @@ def _validate_field_names(self, val):
         try:
             assert len(val) == len(set(val))
         except AssertionError:
-            raise Exception("Field names must be unique")
+            raise Exception("Field names must be unique!")
```

Comment:
Please don't add exclamation marks to exception here and elsewhere, they were only recently removed in https://github.com/jazzband/prettytable/pull/101.

Reply:
This was the version problem i was talking about, I reverted the changes.